compiler/ballerina-backend-llvm/src/main/ballerina/src/compiler_backend_llvm/llvm_bb_body_gen.bal

```diff
@@ -79,14 +93,41 @@ type BbBodyGenrator object {
     }
 
     function genConstantLoadIns(bir:ConstantLoad constLoad) {
-        if !(constLoad.value is int) {
-            error err = error("NotIntErr", message = "invalid value in constLoad");
-            panic err;
+        if (constLoad.value is int) {
+            self.genIntConstLoadIns(constLoad);
+        } else {
+            self.genBoolConstLoadIns(constLoad);
         }
```

Comment:
It would be better to add a `else if` for `boolean` type followed by a `else` case which will handle all the unsupported types (may be panic with a proper error). This will help for debugging in the long run, as we add more types in future.

Reply:
+1. Will add necessary changes